gnome-initial-setup/gis-page-util.c

```diff
@@ -492,3 +491,94 @@ gis_page_util_show_demo_dialog (GisPage *page)
   gtk_window_set_modal (GTK_WINDOW (demo_dialog), TRUE);
   gtk_window_present (GTK_WINDOW (demo_dialog));
 }
+
+static gboolean
+run_dialog_cb (gpointer user_data)
+{
+  GtkDialog *dialog = GTK_DIALOG (user_data);
+  gtk_dialog_run (dialog);
+  return G_SOURCE_REMOVE;
+}
+
+static void
+on_reformatter_exited (GTask  *task,
+                       GError *error)
+{
+  GisPage *page = GIS_PAGE (g_task_get_source_object (task));
+  GisDriver *driver = page->driver;
```

Comment:
You don't really need to declare a variable for something that's only used once and in the next line.

Reply:
ok.